longshot/main.tf

```diff
@@ -0,0 +1,24 @@
+variable "papertrail_prod_destination" {}
+variable "papertrail_qa_destination" {}
+
+resource "heroku_pipeline" "longshot" {
+  name = "longshot"
+}
+
+module "longshot-qa" {
+  source = "application"
+
+  name           = "longshot-qa"
+  host           = "longshot-qa.dosomething.org"
+  pipeline       = "${heroku_pipeline.longshot.id}"
+  pipeline_stage = "staging"
+
+  email_name    = "Longshot QA"
+  email_address = "devops@dosomething.org"
```

Comment:
who checks this email?

Reply:
This is an email group that currently consists of Matt, Morgan, Gleb, and myself. Dave, did we add you to this? If not, we should.

Reply:
I believe the `devops@dosomething.org` address forwards to Matt, Sena, and Morgan. It actually may be worth removing this environment variable in Longshot, since I believe this "from" address is actually set by a value stored in the application's database in most cases.

Reply:
Oh, @sheyd beat me to it! Nope, I don't think I'm on this group but may just not have noticed.

Reply:
@DFurnes we still need an email set in the environment because that is what the password reset via Laravel pulls from

Reply:
💡

Reply:
@katiecrane Ahhh, that explains it! 🙇

Reply:
Thank you! and hey Kathie! Might be worth adding a comment with Katie's explanation.